prod/bsc/feeds.yaml

```diff
@@ -9883,7 +9883,37 @@ CPI-US-3months:
         params:
           months: 3
 
-# OPTIONS CONFIG
+# yearn.finance
+
+YEARN-TEST:*-ETH:
+  discrepancy: 1
+  precision: 6
+  inputs:
+    - fetcher:
+        name: YearnVaultTokenPrice
+        params:
+          address: "0x0E928957572875954687BcA992fBA397642d0511"
+          network: eth-ropsten
+      calculator:
+        name: YearnTransformPrice
+        params:
+          tsym: ETH
+
+YEARN-FI:*-ETH:
+  discrepancy: 1
+  precision: 6
+  inputs:
+    - fetcher:
+        name: YearnVaultTokenPrice
+        params:
+          address: "0x619B772d63399Ac0Cb70EA6ceFF25a8Fc81faE81"
+          network: eth-mainnet
+      calculator:
+        name: YearnTransformPrice
+        params:
+          tsym: ETH
+
+# options
 
 OP:ETH:
   discrepancy: 1
```

Comment:
gotta align these OP to match the new asterisk and calculator stuff.

Reply:
Leave it to me